README.md

```diff
@@ -145,16 +188,27 @@ code, include a script or makefile that will do the job.
 
 ## A2.5
 
-«replace this with your answer»
+run main.cpp with sentences.txt as a command line argument:
```

Comment:
It's a nice piece of code, but a couple of things are holding it back.

1. It doesn't have any tests. Yes, it does take input from a file and spray messages to the console, but that doesn't test the code; at best it lets some hapless human do it. A test knows the expected result from each call, and reports if the actual result is different.

# -2

Second, rather than use the state table as a data structure, you wrote it all in executable code. Normally that's a 6 point deduction, but your code actual is a representation of the states, so it's -3

# -3